app/views/lectures/show.html.erb

```diff
@@ -73,8 +76,11 @@
 
   </div>
   <% if !@current_user.is_student %>
+    <div class="tab-pane fade" id="enrollmentKey" role="tabpanel" aria-labelledby="enrollmentKey-tab-tab">
+      <h1 class="padded-text">Enrollment Key: <%= @lecture.enrollment_key %></h1>
+    </div>
```

Comment:
Again: what if there is no Key? Maybe a shortcut for adding a key would be nice in that case :)